<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>744</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="lEngine">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>117</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Engine</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEngine">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPiecesStyle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lPiecesStyle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>198</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Pieces style</string>
   </property>
  </widget>
  <widget class="QLabel" name="lGameControl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Game control</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPlayerTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>279</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Player time</string>
   </property>
  </widget>
  <widget class="QLabel" name="lEngineTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>319</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Engine time</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPlayerTime">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEngineTime">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPondering">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>441</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Pondering</string>
   </property>
  </widget>
  <widget class="QLabel" name="lOtherSettings">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Other settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="lGeneralSettings">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string notr="true">General settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPlayerName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>78</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Player name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edPlayerName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lInterfaceLanguage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>676</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Interface language</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbInterfaceLanguage">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>670</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLabel" name="lEngineProfile">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>157</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Engine profile</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbEngineProfile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbCoordinateMoveInput">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>542</y>
     <width>491</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Coordinate move input</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDrawCoordinates">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>478</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw coordinates on board sides</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDrawMoveArrow">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>445</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw last move arrow</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAutoSaveGames">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>610</y>
     <width>501</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto save games</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbQuickSingleMoveSelection">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>576</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Allow quick selection of single moves</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbShowMoveHints">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>511</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show move hints</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edPlayerName</tabstop>
  <tabstop>cmbEngine</tabstop>
  <tabstop>cmbEngineProfile</tabstop>
  <tabstop>cmbPiecesStyle</tabstop>
  <tabstop>cmbPlayerTime</tabstop>
  <tabstop>cmbEngineTime</tabstop>
  <tabstop>cbPondering</tabstop>
  <tabstop>cbDrawMoveArrow</tabstop>
  <tabstop>cbDrawCoordinates</tabstop>
  <tabstop>cbShowMoveHints</tabstop>
  <tabstop>cbCoordinateMoveInput</tabstop>
  <tabstop>cbQuickSingleMoveSelection</tabstop>
  <tabstop>cbAutoSaveGames</tabstop>
  <tabstop>cmbInterfaceLanguage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
